### ISSUE TITLE
Problem updating

### ISSUE THREAD

**Issue by alejandrofm** [netdata/netdata#6088]
netdata -v returns **netdata v1.14.0-rc0-20-nightly**
updater returns 1.15 but says **Newest version is already installed**
Ubuntu 18.04, no problem with 3 identical machines, maybe I installed them differently?
```
x@instance-3:~$ netdata -v
**netdata v1.14.0-rc0-20-nightly**

x@instance-3:~$ bash <(curl -Ss https://my-netdata.io/kickstart.sh)
 --- Re-installing netdata... ---
$'[\E[2m/home/x\E(B\E[0m]$ \E[1m\E[33m' sudo /etc/cron.daily/netdata-updater -f $'\E(B\E[0m\\n' Wed May 22 18:18:53 UTC 2019 : INFO:  Running on a terminal - (this script also supports running headless from crontab)
e8df9e77d7f8ab19321afa6014eaf6265ece65b51143c23f21f69682791e20fc *./latest-version.txt
7c74b45d996a0f6b26c652848b533cd1620da4078ce84611745348ad66758020 *./netdata-latest.gz.run
1a192abbad5461736854b91055d2324bc6b491c992eb49559a16161d1ff1c7b7 *./netdata-latest.tar.gz
7c74b45d996a0f6b26c652848b533cd1620da4078ce84611745348ad66758020 *./**netdata-v1.15.0-2-ge12af64d.gz.run**
1a192abbad5461736854b91055d2324bc6b491c992eb49559a16161d1ff1c7b7 *./**netdata-v1.15.0-2-ge12af64d.tar.gz**
Wed May 22 18:18:53 UTC 2019 : INFO:  **Newest version is already installed**
 OK  sudo /etc/cron.daily/netdata-updater -f
```
Thank you!

**Comment by It0w** [comment]
quick and **dirty** =

open /etc/cron.daily/netdata-updater

find **if grep "${NETDATA_TARBALL_CHECKSUM}" sha256sum.txt >&3 2>&3; then**
edit it to **if grep "${NETDATA_TARBALL_CHECKSUM}" sha2561sum.txt >&3 2>&3; then**

save and run **/etc/cron.daily/netdata-updater**

**Comment by alejandrofm** [comment]
> quick and **dirty** =
> 
> open /etc/cron.daily/netdata-updater
> 
> find **if grep "${NETDATA_TARBALL_CHECKSUM}" sha256sum.txt >&3 2>&3; then**
> edit it to **if grep "${NETDATA_TARBALL_CHECKSUM}" sha2561sum.txt >&3 2>&3; then**
> 
> save and run **/etc/cron.daily/netdata-updater**

worked, don't know if this is a problem that only happened to me, so I'm not closing the issue, but it's solved here.
Thank you

**Comment by paulkatsoulakis** [comment]
Hello @alejandrofm, thank you for your report.

  I am truly sorry for this unfortunate event. We had to re-release due to a major fix that we had to bring on board to 1.15.0, but apparently your updater run before i manage to clean up the versions that came out last night after releasing.
  The fix proposed by @It0w is a bit dirty, but also did the trick.  Alternatively you could just wait tonight's nightly that will produce the next version after 1.15.0 and would cover also your current version which is 1.15.0-2.  Please don't forget to modify the file you changed, back to the original form so that it can continue to work on the next iteration. 
  I will keep the issue open as i need to investigate a bit how to secure future releases from such incidents.

P.S. Thanks a lot for jumping in to help @It0w, i love it that our community is so involved on our effort!

**Comment by alejandrofm** [comment]
@paulkatsoulakis after the change I updated via:
bash <(curl -Ss https://my-netdata.io/kickstart.sh)
So the cron file was recreated and my change replaced by the original line, no need to edit back manually.
Thank you too!

**Comment by paulkatsoulakis** [comment]
> @paulkatsoulakis after the change I updated via:
> bash <(curl -Ss https://my-netdata.io/kickstart.sh)
> So the cron file was recreated and my change replaced by the original line, no need to edit back manually.
> Thank you too!

You 're welcome and again my apologies for the bad experience, we 're working towards eliminating discomforts and this should not have happened to you

**Comment by cakrit** [comment]
The easiest hack to fetch the latest version regardless of checksums is to modify the`NETDATA_TARBALL_CHECKSUM` in `/etc/netdata/.environment`. e.g. add a letter to that long checksum, so it doesn't match.

**Comment by paulkatsoulakis** [comment]
I am afraid i could not reproduce the use case. I am suspecting it is tightly related to the fact we re-released our 1.15.0 and at that time something went terribly wrong. It may also be the case that the installed updater was buggy, as i noticed there was a fixed bug on 1.14.0-rc0-20 version.

At any case, i do not consider this resolved but rather it cannot be reproduced.

@alejandrofm by all means do get in touch with us on a new issue if you notice anything suspicious in the update process of netdata. 
  We are constantly in the process of better monitoring the installers performance throughout all supported distributions, we are also preparing our binary distribution repositories to ease our user's lives, but like i have said before we are heavily dependent on our community. You guys have all the nit picking we are looking for, so please do let us know on any problems you face.

See ya!
